agent/commands/run.go

```diff
@@ -94,17 +106,18 @@ func Run() {
 		logrus.Infof("Window check connection time is %.2f hour(s)", cfg.WindowConnectedTime.Hours())
 		connectionUptimeService.SetWindowPeriod(cfg.WindowConnectedTime)
```

Comment:
why are lines 98-107 executed in a loop?
if agent can't establish connection to server should it performed all these actions on each connect attempt?

Reply:
it's to update values from the config file. I'll create a separate task for config management to update configuration when pmm-agent gets a SIGHUP signal or `/reload/` call.